fele-client-service/restful-api/handlers/fele-client.handler.js

```diff
@@ -0,0 +1,26 @@
+const { createNetwork } = require('../../client-api/scripts/network')
+
+const createNetworkHandler = async(req, res) => {
+    const networkName = req.query.networkName
+    await createNetwork(JSON.stringify(req.body), networkName)
+    res.send(
+        {
+            status: "200 OK", 
```

Comment:
Can we send the status as a number ?

Reply:
The status will be a number. I just added that to visualize. Once we finalize the request and response body, we can change the structure of json

Reply:
Sure